src/Service/PaymentService.php

```diff
@@ -69,14 +71,13 @@ public function create($param, $correlationId)
             return $response;
             
         } catch (Exception $e) {
-            // @todo catch and log error
-            // return http status code
-            // throw new RuntimeException('cko Payment error : ' . $e->getMessage());
+
+            CkoLogger::log($e->getMessage(), "cko create payment", "checkout.create.payment.error", $correlationId, "Error");
```

Comment:
CorrelationID should be unique for a single request, so we can set it once (for example in the controller) as:

```CkoLogger::setCorrelationId('XXX');```

Which means we don't need to pass it for every log statement.
Please look at the cloud plugin for similar pattern.